package.json

```diff
@@ -66,9 +66,14 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "testem ci",
-    "lint-dts": "tslint sweetalert2.d.ts --format verbose",
-    "check-dts": "tsc sweetalert2.d.ts && npm run lint-dts",
+    "start": "gulp watch",
+    "fix:lint": "standard --fix",
+    "test": "npm run build && npm run check",
+    "build": "gulp build",
+    "check": "npm run check:bundlesize && npm run check:tests && npm run check:ts",
```

Comment:
indicates all the required checks (except linting which is covered by `gulp build`)

Reply:
I'm not sure if we need this. I personally prefer to be explicit in the Travis configuration and rather run 3 steps instead 1 with the unclear name.

Reply:
Your statement seems to indicate two distinct issues

> prefer to be explicit in the Travis configuration

I think it's better to be explicit here (unless unit tests are the only check they should care about). Contributors don't run the Travis configuration in local development. This way they are likely to run all the required checks before pushing (without even thinking about it). 

> with the unclear name

I find `check` to be pretty clear, and gives us a category of package script. 

(Note that `check` currently is not in the `.travis.yml`.. `test` is. `check` is currently only used in the `test` script 2 lines above, so, as is, I think it's unlikely that anyone will be confused by this. Is there something else that "check" could be interpreted to mean in this context of `.travis.yml`?)

What about `test:bundlesize`, `test:unit`, `test:typescript`?